cmd/eval_test.go

```diff
@@ -273,7 +273,36 @@ func testEvalWithInvalidSchemaFile(t *testing.T, input string, query string, sch
 	return err
 }
 
-func TestEvalWithJSONSchemaFile(t *testing.T) {
+func testReadParamWithSchemaDir(t *testing.T, input string, query string, inputSchema string) error {
+	files := map[string]string{
+		"input.json":                          input,
+		"schemas/default-input-schema.json":   inputSchema,
+		"schemas/kubernetes/data-schema.json": inputSchema,
+	}
+
+	var err error
+	test.WithTempFS(files, func(path string) {
+
+		params := newEvalCommandParams()
+		params.inputPath = filepath.Join(path, "input.json")
```

Comment:
Is this setting relevant for the test case...? I'm not sure I see how.

Reply:
It's not quite relevant but since a schema is being provided at `params.schemaPath`, it makes sense that the `inputPath` also has the input json file. Otherwise, an input schema without an input for eval isn't realistic.

Reply:
Ah, of course! Thanks.